### PR TITLE
Clarify single hash function requirements.

### LINKED ISSUE #26: Clarify MUSTs
From Rob Sayre:

~~~
I'm not sure how violations of these requirements would result in poor interoperability:

 Clients which import external keys TLS MUST NOT use these keys for
 any other purpose. Moreover, each external PSK MUST be associated
 with at most one hash function.

These seem like aspirational security goals. It would be better to describe the consequences of violating these conditions.
~~~

We should note the latter is a requirement from Section 4.2.11 of RFC8446. Let's also fix the typo "import external keys *for* TLS..."

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #26.

cc @sayrer

### CODE REVIEW

File: draft-ietf-tls-external-psk-importer.md [hunk 1]
```diff
@@ -99,8 +98,10 @@ identities if not imported correctly. Endpoints may incrementally deploy PSK imp
 by offering non-imported keys for TLS versions prior to TLS 1.3. Non-imported and imported PSKs
 are distinct since their identities are different on the wire. See {{rollout}} for more details.
 
-Clients which import external keys TLS MUST NOT use these keys for any other purpose.
-Moreover, each external PSK MUST be associated with at most one hash function.
+Clients which import external keys for TLS MUST NOT use these keys for any other
+purpose. Moreover, each external PSK MUST be associated with at most one hash function,
```
Comment (ekr) [inline draft-ietf-tls-external-psk-importer.md]: Does this mean you must not use the key you are feeding into the importer or the key you derive from the importer?

Comment (chris-wood) [reply draft-ietf-tls-external-psk-importer.md]: The former, though that may be a bit too strong of a requirement.

Comment (ekr) [reply draft-ietf-tls-external-psk-importer.md]: As I understand the technical requirement it's that you must only use it with HKDF with that hash and a unique label. So if, for instance, you were to HKDF with a label that guaranteed non-collision with importers, that would be OK, right

Comment (chris-wood) [reply draft-ietf-tls-external-psk-importer.md]: That's right, and that's what the security considerations discuss. (Are you looking for a change in text here?)

**Comment by chris-wood** [comment]
I made a clarifying change on #28 -- closing in favor of that PR!

**Pull request closed without merging**